db/migrate/20170402211133_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[5.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
```

Comment:
It's a good practice to enforce as strict constraints on data as possible. Not null and unique constraints would be great here.